velox/expression/fuzzer/ExpressionFuzzerTest.cpp

```diff
@@ -132,6 +132,15 @@ int main(int argc, char** argv) {
       "st_geomfrombinary",
       "st_astext",
       "st_asbinary",
+      "st_relate",
+      "st_contains",
+      "st_crosses",
+      "st_disjoint",
+      "st_equals",
+      "st_intersects",
+      "st_overlaps",
+      "st_touches",
+      "st_within",
```

Comment:
It seems that this PR is based on https://github.com/facebookincubator/velox/pull/13330. Can you also add overlay functions here since they can geometry types as the input